blockly-core/core/ui/function_editor.js

```diff
@@ -192,6 +216,15 @@ Blockly.FunctionEditor.prototype.renameParameter = function(oldName, newName) {
   });
 };
 
+Blockly.FunctionEditor.prototype.changeParameterType = function(name, newType) {
+  this.orderedParamIDsToBlocks_.forEach(function(blockXML, paramID, linkedMap) {
+    if (blockXML.firstElementChild &&
+        blockXML.firstElementChild.textContent === name) {
+      blockXML.childNodes[1].firstElementChild.textContent = newType;
```

Comment:
I'd like to eventually move this to use closure tools' xpath since this is pretty brittle